server/src/main/scala/com/faleknatalia/cinemaBookingSystem/JsonSupport.scala

```diff
@@ -1,18 +1,24 @@
 package com.faleknatalia.cinemaBookingSystem
 
-import java.time.LocalDateTime
+import java.net.URI
+import java.time.ZonedDateTime
 
 import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
 import com.faleknatalia.cinemaBookingSystem.cinemahall.{CinemaHall, Seat}
 import com.faleknatalia.cinemaBookingSystem.movie.{AddMovie, Movie, ScheduledMovie, ScheduledMovieDto}
 import spray.json.{DefaultJsonProtocol, JsString, JsValue, JsonFormat}
 
 trait JsonSupport extends SprayJsonSupport with DefaultJsonProtocol {
-  implicit val localDateTimeFormat: JsonFormat[LocalDateTime] = new JsonFormat[LocalDateTime] {
-    override def read(json: JsValue): LocalDateTime = ???
+  implicit val ZonedDateTimeFormat: JsonFormat[ZonedDateTime] = new JsonFormat[ZonedDateTime] {
+    override def read(json: JsValue): ZonedDateTime = ???
```

Comment:
kiedys bedzie trzeba usunac to `???` - w produkcyjnym kodzie tego nie powinno byc